dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml

```diff
@@ -46,7 +46,7 @@
     = render partial: 'csp_pre_course_reading'
 
 %p
-  If you have any accessibility needs or food allergies, please let the workshop organizer know so they can make appropriate accommodation plans and answer any of your questions. You can reach them at
+  If you have any accessibility needs or food allergies, please contact the workshop organizer to discuss appropriate accommodation plans and to answer any of your questions. You can reach them at
   = mail_to(@workshop.organizer.email) + '.'
```

Comment:
It's intentional that this language change applies to all of our workshop types, not just the new one.